frontend/src/components/TodoList.tsx

```diff
@@ -2,13 +2,20 @@ import React from "react";
 import { Todo } from "../models/todo.model";
 
 import TodoListItem from "./TodoListItem";
-
-export default function TodoList(props: any) {
+interface todoList {
```

Comment:
Interfaces that define the shape of a components props are usually named like this: ComponentName + Props

e.g. We used ` InputProps` for the `Input` component.  Change this to `TodoListProps`.